### PR TITLE
[FIX] Streamline log-level flags, allow overwrite with env variable

### PR DESCRIPTION
**Pull request by RandomByte**
The Yargs defaulting of the log-level argument let to an unwanted
enforcing of log-level "info", which could not be overwritten using the
UI5_LOG_LVL environment variable.

If a user does not specify any log-level related flags, no log level
should be set by the CLI.

This PR also fixes a minor bug in the output of `ui5 tree`

### CODE REVIEW

File: lib/cli/base.js [hunk 1]
```diff
@@ -13,18 +13,24 @@ export default function(cli) {
...
 			type: "string"
 		})
-		.option("x-perf", {
-			describe: "Outputs performance measurements",
+		.option("verbose", {
+			describe: "Enable verbose logging.",
+			default: false,
+			type: "boolean"
+		})
+		.option("silent", {
```
Comment (flovogt) [inline lib/cli/base.js:27]: should we order the log options according to the log level ordering?

File: lib/cli/base.js [hunk 2]
```diff
@@ -13,18 +13,24 @@ export default function(cli) {
...
+			describe: "Disable all log output.",
+			default: false,
+			type: "boolean"
+		})
+		.option("perf", {
```
Comment (flovogt) [inline lib/cli/base.js:32]: so `perf` is no longer experimental?

**Review by flovogt — CHANGES_REQUESTED** [review]

File: lib/cli/base.js (hunk 2 above)
Comment (RandomByte) [reply lib/cli/base.js:32]: Yes. The CLI option was only used in the `ui5 tree` command for some benchmarking. Now it also sets the corresponding log level, which makes sense I think.

File: lib/cli/base.js (hunk 1 above)
Comment (RandomByte) [reply lib/cli/base.js]: Done

**Review by flovogt — APPROVED** [review]

**Pull request merged**